### PR TITLE
Add libomxil_bellagio

### PR DESCRIPTION
**Pull request by lovedheart**
Bellagio is an opensource implementation of the Khronos OpenMAX Integration Layer
API to access multimedia components.

## Addtional information
Mention things we might need to know. Like:

Works properly:
- [ ] x86_64 (no tested)
- [x] armv7l

### CODE REVIEW

File: packages/libomxil_bellagio [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'package'
+
+class Libomxil_bellagio < Package
+  description 'Bellagio is an opensource implementation of the Khronos OpenMAX Integration Layer
+API to access multimedia components.'
```
Comment (uberhacker) [inline packages/libomxil_bellagio]: Can you please make the description a single line?

File: packages/libomxil_bellagio [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  homepage 'https://sourceforge.net/projects/aa-project/'
+  version '0.9.3'
+  source_url 'https://sourceforge.net/projects/omxil/files/omxil/Bellagio%200.9.3/libomxil-bellagio-0.9.3.tar.gz'
+  source_sha256 '593c0729c8ef8c1467b3bfefcf355ec19a46dd92e31bfc280e17d96b0934d74c'
+
+  def self.build
+    system "./configure","--prefix=#{CREW_PREFIX}","--libdir=#{CREW_LIB_PREFIX}"
+    system "make", "CFLAGS=-Wno-error=switch"    # only -j1 possible (tested on armv7l)
```
Comment (uberhacker) [inline packages/libomxil_bellagio]: Should we just go ahead and add the `, "-j1"` option here for all architectures?

Comment (lovedheart) [reply packages/libomxil_bellagio]: I am not sure if that must be down on x86_64. But with -j1 is better for all. It does not take much time. (less than 1 min)

Comment (uberhacker) [reply packages/libomxil_bellagio]: Do you need the `-j1` option for `armv7l` or does the current line work?

Comment (lovedheart) [reply packages/libomxil_bellagio]: current line does not work.  Plz  put -j1 for armv7l

Comment (uberhacker) [reply packages/libomxil_bellagio]: Can you please add it?  It's your PR. ;)

**Pull request merged**